src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -975,7 +980,15 @@ public boolean isInteresting(GerritTriggeredEvent event) {
                          && p.isInteresting(changeBasedEvent.getChange().getProject(),
                                             changeBasedEvent.getChange().getBranch(),
                                             changeBasedEvent.getChange().getTopic())) {
-
+                        
+                        if (event instanceof WipStateChanged && !isServerSupportFeature(event, GerritVersionChecker.Feature.wipStateChanged)) {
```

Comment:
Seems redundant at first glance. If the server doesn't support it you shouldn't get the event?